app/portainer/views/stacks/create/createStackController.js

```diff
@@ -36,6 +40,8 @@ angular
       formValidationError: '',
       actionInProgress: false,
       StackType: null,
+      yamlValidationError: '',
+      yamlUploadError: '',
```

Comment:
this variable name is confusing as it let think the error would come from upload, but we are using it for another validation of yaml file on upload, to keep consistent with the other one I would use something like `uploadYamlValidationError` and maybe `editorYamlValidationError`